lib/savon/request_logger.rb

```diff
@@ -29,13 +29,13 @@ def log?
     def log_request(request)
       logger.info  { "SOAP request: #{request.url}" }
       logger.info  { headers_to_log(request.headers) }
-      logger.debug { body_to_log(request.body) }
+      logger.info { body_to_log(request.body) }
     end
 
     def log_response(response)
-      logger.info  { "SOAP response (status #{response.code})" }
-      logger.debug { headers_to_log(response.headers) }
-      logger.debug { body_to_log(response.body) }
+      logger.info { "SOAP response (status #{response.code})" }
+      logger.info { headers_to_log(response.headers) }
+      logger.info { body_to_log(response.body) }
```

Comment:
Can we make this a single line log? Structure the message body in JSON like object?

Reply:
this is standard across most libraries as headers and body are separate information. and people usually use them separately, for example: when checking successful rate, checking header/response code is enough. We can associated them with rails tags, so should be ok.

Reply:
https://github.com/Go-Shippit/allied-express-sdk-ruby/pull/10
for your assurance that this PR was only logging body in one line too, without merging with headers or status code.